docs/zeta_integration.adoc

```diff
@@ -0,0 +1,85 @@
+= Zeta Integration Design
+
+== Architecture Overview
+
+image::images/zeta_integration_high_level.jpg["highlevel Architecture", width=800, link="zeta_integration_high_level.jpg"]
+
+1. Control/API Network. Alcor server configures the group entries and the buckets in the group entries for each computing node. 
+Each group entry corresponds to a Zeta Gateway Cluster (ZGC). Each bucket in the group entry corresponds to a gateway in ZGC.
+
+2. Tenant Network. The gateway node in ZGC is responsible for delivering the rules to ACA on computing nodes when two VMs communicate for the first time. 
+ACA as the openflow controller will install rules on ovs.
+
+3. Gateway/Direct path. The path between the gateway node and the computing node is called the gateway path whereas VM to VM direct communication is called direct path.
```

Comment:
I have added number 3, renamed default path to gateway for clarity. Gateway and direct path is an important concept. Please add a picture on it.